tests/units/iop/test_iop_parser.py

```diff
@@ -0,0 +1,31 @@
+import xml.etree.ElementTree as ET
+
+from common.parsing.xml_extractors import RequiredFieldNotFoundExtractionError
+from iop.parser import IOPParser
+from pytest import fixture, raises
+
+
+@fixture
+def parser():
+    return IOPParser()
+
+
+def test_doi(shared_datadir, parser):
+    content = (shared_datadir / "just_required_fields.xml").read_text()
+    article = ET.fromstring(content)
+    parsed_article = parser._publisher_specific_parsing(article)
+    assert parsed_article["dois"] == ["10.1088/1674-1137/ac66cc"]
```

Comment:
Since we are going through this process, let's but in `common/jsonschema/scoap3.json` and let's have in the conftest a fixture for all tests with the `scoap3_schema` that reads the `scoap3.json` once. Then in the test we can test the subschema of a field for example:

```python
subschema = schema['properties']['dois']
assert validate(parsed_article['dois'], subschema) is None
```